designguidelines.md

```diff
@@ -0,0 +1,15 @@
+These components are used as the building blocks for creating all types of forms. These forms should be able to be used on all devices ie desktop, mobile and tablet.
+
+## Form rules:
+
+*Labels* - Always label the form component ie a text input, select box to help the user understand it’s meaning.
```

Comment:
I think each item needs to be prepended with `- ` in order for Markdown to recognise it as a list.